wordpress-mobile/WordPress-Android
# Additional tracking information of media upload errors

FluxC PR - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1755

This PR adds 3 additional fields to the tracked media upload error. The purpose is to be able to analyze the various error types (right now we're only tracking the error type that's quite generic (one of the most common errors has literally "GENERIC_ERROR" type). Adding the message and the status code should help us find out if we're missing something during the upload. Let me know if it makes sense. 

To test:
- Simulate an error (for example turn off the data during media upload)
- Check an event is tracked in Logcat
- Check it has the new fields present

PR submission checklist:
